diff --git a/vendor/github.com/go-kivik/kivik/iterator.go b/vendor/github.com/go-kivik/kivik/iterator.go
index f12bf985..4bc7f1cb 100644
--- a/vendor/github.com/go-kivik/kivik/iterator.go
+++ b/vendor/github.com/go-kivik/kivik/iterator.go
@@ -79,6 +79,8 @@ func (i *iter) next() (doClose, ok bool) {
 		return false, false
 	}
 	i.ready = true
+	v := reflect.ValueOf(i.curVal).Elem()
+	v.Set(reflect.Zero(v.Type()))
 	i.lasterr = i.feed.Next(i.curVal)
 	if i.lasterr != nil {
 		return true, false
